new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -306,6 +334,47 @@ public void testDomainInPvUsingWlst() {
       return adminNodePortAccessible(serviceNodePort, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
     }, "Access to admin server node port failed");
     assertTrue(loginSuccessful, "Console login validation failed");
+
+    logger.info("Getting node port for T3 channel");
+    int t3channelNodePort = assertDoesNotThrow(()
+        -> getServiceNodePort(wlstDomainNamespace, adminServerPodName + "-external", "t3channel"),
+        "Getting admin server t3channel node port failed");
+    assertNotEquals(-1, t3ChannelPort, "admin server t3channelport is not valid");
+
+    //create ingress controller
+    Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
+    clusterNameMsPortMap.put(clusterName, managedServerPort);
+    logger.info("Creating ingress for domain {0} in namespace {1}", domainUid, wlstDomainNamespace);
+    createIngressForDomainAndVerify(domainUid, wlstDomainNamespace, clusterNameMsPortMap);
+
+    //deploy application
+    Path archivePath = Paths.get(ITTESTS_DIR, "../src/integration-tests/apps/testwebapp.war");
```

Comment:
Can we add a TestConstant for APPDIR instead of "../src/"

Reply:
We already have a APP_DIR which is pointing to "user.dir" + /src/test/resources/apps
If I had to use that constant it will look like
Paths.get(APP_DIR, "../../../../../../src/integration-tests/apps/testwebapp.war");

Reply:
discussed this with Pani, agreed on no changes needed